### PR TITLE
Ft22

### PR DESCRIPTION
**Pull request by ricky1uphealth**

### CODE REVIEW

File: spec/export/index.md [hunk 1]
```diff
@@ -210,6 +210,7 @@ Retry-After: 120</code></pre></td>
       <td>Returned by the server if the export operation fails.</td>
       <td><pre><code>Status: 500 Internal Server Error
 Content-Type: application/json
+Retry-After: 120
```
Comment (jmandel) [inline spec/export/index.md]: The failed example shouldn't have a retry-after header -- once failed, the request is done.

File: spec/export/index.md [hunk 2]
```diff
@@ -267,27 +268,30 @@ Content-Type: application/json
...
+
   {
     "resourceType": "OperationOutcome",
-    "id": "101",
+    "id": "1",
     "issue": [
       {
         "severity": "error",
-        "code": "deleted",
+        "code": "timeout",
```
Comment (jmandel) [inline spec/export/index.md]: `timeout` is not a valid code. See http://build.fhir.org/valueset-issue-type.html; maybe just use `processing`

File: spec/export/index.md [hunk 3]
```diff
@@ -267,27 +268,30 @@ Content-Type: application/json
...
 - ```Content-Type``` header of ```application/json```
 - The server SHALL return a FHIR OperationOutcome resource in JSON format
-- The choice of when to determine that an export job has failed in its entirety (error status) vs returning a partial success (complete status) is left up to the implementer.
 
 	Example OperationOutcome response body:
 
   ```
+  Status: 500 Internal Server Error
+  Content-Type: application/json
+  Retry-After: 120
```
Comment (jmandel) [inline spec/export/index.md]: The failed example shouldn't have a retry-after header -- once failed, the request is done. Is there a reason we're including an example here when there's already an example in the table above? Would be better to cut this, unless it's distinct.

Comment (ricky1uphealth) [reply spec/export/index.md]: Yup i agree, we should remove it. I just committed that here https://github.com/HL7/bulk-data/pull/32/commits/72b3f10261b3495259313fe1b9237e8044526215

File: spec/export/index.md (hunk 2 above)
Comment (ricky1uphealth) [reply spec/export/index.md]: fixed here in the table https://github.com/HL7/bulk-data/pull/32/commits/48e1eec7c20d54e9e817ff65bd86aa370690a4c5 (the example i agreed to delete bc we have the dupe of the example in the table)

File: spec/export/index.md (hunk 1 above)
Comment (ricky1uphealth) [reply spec/export/index.md]: fixed https://github.com/HL7/bulk-data/pull/32/commits/e39e47500036c21b01b50a4033cf054ee69f7437

**Comment by ricky1uphealth** [comment]
@jmandel  this is ready to review. thanks for the feedback.

**Pull request merged**